src/App.js

```diff
@@ -1,22 +1,16 @@
 import { Component } from 'react';
-import PropTypes from 'prop-types';
+import { ButtonPanel, Display } from './components';
 import './App.css';
 // eslint-disable-next-line react/prefer-stateless-function
 class App extends Component {
   render() {
-    const { title } = this.props;
     return (
       <div className="App">
-        <h1>{title}</h1>
+        <Display />
+        <ButtonPanel />
```

Comment:
- [ ] Please wrap these components in a `React.Fragment` short syntax instead. To avoid adding an unnecessary div node.